frontend/server/src/main/java/org/pytorch/serve/wlm/BatchAggregator.java

```diff
@@ -56,8 +62,8 @@ public BaseModelRequest getRequest(String threadName, WorkerState state)
     }
 
     public void sendResponse(ModelWorkerResponse message) {
+        boolean jobDone = true;
```

Comment:
this variable name is a bit confusing, outside of the context of streaming - the job is not done yet, maybe streamcomplete or something of the sort would be cleaerer

Reply:
The most use cases are non-streaming. They only  require one single message retrieving for a batch of jobs. Var "jobDone" is used to reflect if the message retrieving is completed for a batch of jobs.